tests/testsuite/help.rs

```diff
@@ -48,7 +48,9 @@ fn z_flags_help() {
     // Test that the output of `cargo -Z help` shows a different help screen with
     // all the `-Z` flags.
     cargo_process("-Z help")
-        .with_stdout_contains("    -Z unstable-options    -- Allow the usage of unstable options")
+        .with_stdout_contains(
+            "    -Z allow_features                 -- Allow *only* the listed unstable features",
```

Comment:
```suggestion
            "    -Z allow_features[..]-- Allow *only* the listed unstable features",
```

The `[..]` matches anything, and this can help avoid needing to update the test if the spacing changes.